### PR TITLE
feat: add reload table button

### LINKED ISSUE #222: Logs widget automatic refresh with subscriptions

### PR DESCRIPTION
**Pull request by roshaans**
Adds a button to refresh the logs from indexer logs page.

### CODE REVIEW

**Review by morgsmccauley — APPROVED** [review]

File: frontend/src/components/Logs/IndexerLogs.jsx [hunk 1]
```diff
@@ -204,8 +204,19 @@ const IndexerLogsComponent = () => {
...
+  };
+
+  useEffect(() => {
+    initializeTable();
   }, []);
 
+  const reloadData = () => {
+    indexerLogsRef.current.innerHTML = "";
+    setTimeout(() => {
+      initializeTable();
```
Comment (darunrs) [inline frontend/src/components/Logs/IndexerLogs.jsx:216]: Would this reset the pagination?

File: frontend/src/components/Logs/LogButtons.jsx [hunk 2]
```diff
@@ -29,6 +29,7 @@ const LogButtons = ({
   setHeights,
   latestHeight,
   isUserIndexer,
+  reloadData
```
Comment (darunrs) [inline frontend/src/components/Logs/LogButtons.jsx:32]: nit: Should this be called Reload Logs instead?

**Review by darunrs — APPROVED** [review]

**Comment by darunrs** [comment]
Also, this is an awesome feature. I'm very happy to see it!

File: frontend/src/components/Logs/IndexerLogs.jsx (hunk 1 above)
Comment (roshaans) [reply frontend/src/components/Logs/IndexerLogs.jsx:216]: It empties the dom element that represents the table, and then re-renders it thus reseting the table. 

if you are on page 6 and reload, you'll be reset back to page 0

**Pull request merged**